torchaudio/csrc/ffmpeg/typedefs.h

```diff
@@ -8,9 +8,9 @@ namespace ffmpeg {
 
 struct SrcStreamInfo {
   AVMediaType media_type;
-  const char* codec_name = NULL;
```

Comment:
bug fix

keeping this NULL caused segmentation fault if a source stream is subtitle.